Resources/public/views/recommendations.html.twig

```diff
@@ -0,0 +1,31 @@
+<div id="recommendation-{{ recommendationId }}" class="ez-recommendation">
+    {{ 'Loading recommended items...'|trans }}
+</div>
+
+<script id="recommendation-template-{{ recommendationId }}" type="text/x-handlebars-template">
+    {{ template|raw }}
+</script>
+
+<script>
+    (function() {
+        var recoClient{{ recommendationId }} = new EzRecoRestClient({
+            scenarioId: '{{ scenarioId }}',
+            locationId: '{{ locationId }}',
+            limit: '{{ limit }}',
+            restUrl: '/recommendations/fetch' // TODO: dynamically replace with path from routing.yml
```

Comment:
It would be nice if `restUrl` could contain path loaded from `routing.yml`.
In this case user can define his own rest url path (in `routing.yml`).
I'm gonna implement it later.